pkg/imagebuilder/content.go

```diff
@@ -41,7 +41,11 @@ func New(nodeZeroIP string) *ConfigBuilder {
 		fmt.Errorf("Error marshalling cluster params into json: %w", err)
 	}
 
-	infraEnvParams := manifests.CreateInfraEnvParams()
+	infraEnvParams, err := manifests.CreateInfraEnvParams()
+	if err != nil {
+		fmt.Errorf("Error building infra env params: %w", err)
```

Comment:
What's this for?

Reply:
The error that propagates from `nmstateconfig.validateNMStateConfigAndInfraEnv`

Reply:
My point was about the reason of `fmt.Errorf` without using the returned value (an `error`). Regardless of the source of the error, I was expecting some kind of error management but AFAICS this has no effect